examples/fc_config.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	_ "os"
```

Comment:
os is not used in this file. So this example will give a compilation error. Fix this.